.travis.yml

```diff
@@ -18,14 +18,11 @@ cache:
   directories:
   - node_modules
 before_install:
-- npm install -g npm@3
-- npm --version
+- npm install -g npm@3 
```

Comment:
I'm not sure if yarn needs it but I figured I'd leave it in so that the check script can ensure we're all on at least npm@3 in case someone does an accidental `npm install` or uses `npm` primarily.

I don't actually know why this was required but I think @muffinresearch would know more.